### DISCUSSION TITLE
Error: SendKey Couldn't identify keystroke

### DISCUSSION THREAD

**Discussion by arseniolocsin** [rsjaffe/MIDI2LR#573] [Problems]
I updated MIDI2LR to 5.2.0.0 because I can't switch profiles after updating to Lightroom 11. I restarted my Windows 10 machine and installed the update. When I select the Layer B button and then select a button to change the profile, I see the following error:

SendKey couldn't identify keystroke. Message from plugin was *SendKey 0\n*.

How do I get rid of this error?

**Comment by rsjaffe** [comment]
That error message appears when you press a button assigned to send a keystroke and nothing was assigned to that keystroke number. Are you sure that the button is assigned to "change profile"? Check your current profile--you may have accidentally changed the assignment.

**Comment by arseniolocsin** [comment]
This is what I have for the profile option. Where is change profile?
![image](https://user-images.githubusercontent.com/35786050/144889563-879fd7f9-dc2b-407b-926a-94573f84f93c.png)

**Comment by rsjaffe** [comment]
That is what I meant by change profile: you're changing to a different profile. Which key are you pressing and what is assigned to it? (again, check the actual assignment, not what you remember it being).
